.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "root": true,
+  "env": {
+    "es2020": true,
+    "browser": true,
+    "node": true,
```

Comment:
no need for node, right?

Reply:
Yes, it's required, for example for `scripts/build.js` - we have node scripts.